### ISSUE TITLE
Experimental graybox quantum control. (arXiv:2206.12201v1 [quant-ph])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#20285]
Experimental graybox quantum control. (arXiv:2206.12201v1 [quant-ph])<br>
https://ift.tt/4FVMX9L<br>
<p>Understanding and controlling engineered quantum systems is key to developing practical quantum technology. However, given the current technological limitations, such as fabrication imperfections and environmental noise, this is not always possible. To address these issues, a great deal of theoretical and numerical methods for quantum system identification and control have been developed. These methods range from traditional curve fittings, which are limited by the accuracy of the model that describes the system, to machine learning methods, including neural networks, which provide efficient control solutions but do not provide control beyond the output of the model, nor insights into the underlying physical process. Here we experimentally demonstrate a "graybox" approach for characterizing and controlling quantum devices. We report superior performance over model fitting, while generating unitaries and Hamiltonians which are inaccessible with machine learning techniques. Our approach combines prior physics knowledge with high accuracy machine learning and is effective with any problem where the required controlled quantities are not directly accessible from the training set. This method naturally extends to time-dependent and open quantum systems, with applications in quantum noise spectroscopy and cancellation.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
